lib/workers/repository/index.ts

```diff
@@ -17,10 +17,13 @@ import { ensureOnboardingPr } from './onboarding/pr';
 import { extractDependencies, updateRepo } from './process';
 import { ProcessResult, processResult } from './result';
 import { printRequestStats } from './stats';
+import type { SshSocket } from '../global/ssh_socket';
+import * as hostRules from '../../util/host-rules';
 
 // istanbul ignore next
 export async function renovateRepository(
   repoConfig: RenovateConfig,
+  sshSocket: SshSocket | undefined = undefined,
```

Comment:
```suggestion
  sshSocket: SshSocket | null = null,
```
we prefer null over undefined. 🤗